packages/orchestrator-ui-components/src/components/WfoSettings/WfoFlushSettings.tsx

```diff
@@ -11,14 +11,19 @@ import {
 } from '@elastic/eui';
 import { EuiComboBoxOptionOption } from '@elastic/eui/src/components/combo_box/types';
 
-import { ToastTypes } from '../../contexts';
-import { OrchestratorConfigContext } from '../../contexts/OrchestratorConfigContext';
-import { useToastMessage } from '../../hooks';
-import { useCacheNames } from '../../hooks/DataFetchHooks';
+import { OrchestratorConfigContext, ToastTypes } from '@/contexts';
+import { useCacheNames, useSessionWithToken, useToastMessage } from '@/hooks';
 
-const clearCache = async (apiUrl: string, settingName: string) => {
+const clearCache = async (
+    apiUrl: string,
+    settingName: string,
+    accessToken: string = '',
+) => {
     const response = await fetch(apiUrl + `/settings/cache/${settingName}`, {
         method: 'DELETE',
+        headers: {
+            Authorization: accessToken ? `Bearer ${accessToken}` : '',
```

Comment:
this could be better done with `useQueryWithFetch` that addes the authorization header, so its not necessary everywhere.

Reply:
Fair enough. I think this should be a separate ticket where we centralise all calls that we do in one place.